### PR TITLE
New add colloid method

### PR DESCRIPTION
**Pull request by Koppeprojects**
To the current state it is nor easily possible to place colloids one by one on precise places and orientations. This pull request adds a method for this and rewrites the original way of adding colloids. Be sure to only add colloids one by one if no existing colloid type is given a new radius. Is this condition reasonable. It is there to be compatible with existing colloid radius handling.

### CODE REVIEW

File: swarmrl/engine/espresso.py [hunk 1]
```diff
@@ -59,6 +59,13 @@ def _vector_from_angles(theta, phi):
     )
 
 
+def _angles_from_vector(director):
```
Comment (christophlohrmann) [inline swarmrl/engine/espresso.py:62]: move this to utils

Comment (christophlohrmann) [reply swarmrl/engine/espresso.py:62]: the random angles, to

**Review by christophlohrmann — CHANGES_REQUESTED** [review]
looks good. small comment and we can merge

**Review by SamTov — DISMISSED** [review]
I think it looks good and have no additional comments to Christoph.

**Comment by christophlohrmann** [comment]
@Koppeprojects can you fix the formatting so we can merge? This is a nice PR, let's not let it go stale

**Review by christophlohrmann — APPROVED** [review]

**Review by christophlohrmann — APPROVED** [review]

**Pull request merged**